<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fiche Client</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
/*	background-image:url(image/Zen-Background-23.jpg);
	background-repeat:none;
	background-color:#97d562;*/

/*	background-image:url(image/Zen-Background-23.jpg);*/
	/*background-repeat:none;*/

/*	background-color:#97d562;
	background: qlineargradient(stop: 0 rgba(250,250,250,1), stop: 1 rgba(151,213,98,1));*/
/*	background: qlineargradient( x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 rgba(250,250,250,255), stop: 1 rgba(151,213,98,255));*/

background: qlineargradient( x1: 100, y1:100, x2: 200, y2: 200,
														stop: 0 rgba(250,250,250,255), stop: 1 rgba(151,213,98,255));


}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_suppr">
       <property name="styleSheet">
        <string notr="true">QPushButton 
{
	background-image:url(image/bg-button_ns.png);
	background-repeat:repeat-xy;
    border-color:#739442;
     border-style: outset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}

QPushButton:pressed 
{
     background-image:url(image/bg-button_s.png);
	background-repeat:repeat-xy;
    border-color:#739442;
     border-style: inset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
 }

QPushButton:hover:!pressed
{
	background-image:url(image/bg-button_h.png);
	background-repeat:repeat-xy;
    border-color:#c2bf48;
     border-style: outset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}

</string>
       </property>
       <property name="text">
        <string>Supprimer client</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../build-JustinePgm-Desktop_Qt_5_4_0_MinGW_32bit-Debug/del.png</normaloff>../build-JustinePgm-Desktop_Qt_5_4_0_MinGW_32bit-Debug/del.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_SAVE">
       <property name="styleSheet">
        <string notr="true">QPushButton 
{
	background-image:url(image/bg-button_ns.png);
	background-repeat:repeat-xy;
    border-color:#739442;
     border-style: outset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}

QPushButton:pressed 
{
     background-image:url(image/bg-button_s.png);
	background-repeat:repeat-xy;
    border-color:#739442;
     border-style: inset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
 }

QPushButton:hover:!pressed
{
	background-image:url(image/bg-button_h.png);
	background-repeat:repeat-xy;
    border-color:#c2bf48;
     border-style: outset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}

</string>
       </property>
       <property name="text">
        <string>Modifier client</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../build-JustinePgm-Desktop_Qt_5_4_0_MinGW_32bit-Debug/save.png</normaloff>../build-JustinePgm-Desktop_Qt_5_4_0_MinGW_32bit-Debug/save.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_retour">
       <property name="styleSheet">
        <string notr="true">QPushButton 
{
	background-image:url(image/bg-button_ns.png);
	background-repeat:repeat-xy;
    border-color:#739442;
     border-style: outset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}

QPushButton:pressed 
{
     background-image:url(image/bg-button_s.png);
	background-repeat:repeat-xy;
    border-color:#739442;
     border-style: inset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
 }

QPushButton:hover:!pressed
{
	background-image:url(image/bg-button_h.png);
	background-repeat:repeat-xy;
    border-color:#c2bf48;
     border-style: outset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}

</string>
       </property>
       <property name="text">
        <string>Retour</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../build-JustinePgm-Desktop_Qt_5_4_0_MinGW_32bit-Debug/Cancel.png</normaloff>../build-JustinePgm-Desktop_Qt_5_4_0_MinGW_32bit-Debug/Cancel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="styleSheet">
        <string notr="true">QPushButton 
{
	background-image:url(image/bg-button_ns.png);
	background-repeat:repeat-xy;
    border-color:#739442;
     border-style: outset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}

QPushButton:pressed 
{
     background-image:url(image/bg-button_s.png);
	background-repeat:repeat-xy;
    border-color:#739442;
     border-style: inset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
 }

QPushButton:hover:!pressed
{
	background-image:url(image/bg-button_h.png);
	background-repeat:repeat-xy;
    border-color:#c2bf48;
     border-style: outset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 8em;
     padding: 7px;
}

</string>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../build-JustinePgm-Desktop_Qt_5_4_0_MinGW_32bit-Debug/Ok.png</normaloff>../build-JustinePgm-Desktop_Qt_5_4_0_MinGW_32bit-Debug/Ok.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_nbVisite">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Nb visites:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="nbVisite_valeur">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_total">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Total:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_cout_total">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_euro">
         <property name="text">
          <string>â‚¬</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView_prestations">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableView 
{
     alternate-background-color: #E2E2E2;
     background: white;
     background-image: url(image/bg-tableview.jpg);
     background-repeat: repeat-xy;

	selection-background-color:#c9ff94;
	selection-color:rgb(0, 0, 0);

}

QHeaderView
{
	font: 75  8pt &quot;MS Shell Dlg 2&quot;;
	color:rgb(255, 255, 255);
}

QHeaderView:section
 {
     background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 #616161, stop: 0.5 #505050,
                                       stop: 0.6 #434343, stop:1 #656565);

     padding-left: 4px;
     border: 1px solid #6c6c6c;

    font: 75  8pt &quot;MS Shell Dlg 2&quot;;
    color:rgb(255, 255, 255);
 }



</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ajoutPrestation">
       <property name="styleSheet">
        <string notr="true">QPushButton 
{
	background-image:url(image/bg-button_ns.png);
	background-repeat:repeat-xy;
    border-color:#739442;
     border-style: outset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 10em;
     padding: 7px;
}

QPushButton:pressed 
{
     background-image:url(image/bg-button_s.png);
	background-repeat:repeat-xy;
    border-color:#739442;
     border-style: inset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 10em;
     padding: 7px;
 }

QPushButton:hover:!pressed
{
	background-image:url(image/bg-button_h.png);
	background-repeat:repeat-xy;
    border-color:#c2bf48;
     border-style: outset;
     border-width: 2px;
     border-radius: 2px;
     font: bold 12px;
     min-width: 10em;
     padding: 7px;
}

</string>
       </property>
       <property name="text">
        <string>Ajout prestation</string>
       </property>
       <property name="iconSize">
        <size>
         <width>15</width>
         <height>15</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_nom">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Nom</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_prenom">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>PrÃ©nom</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_age">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Age</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_mobile">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Mobile</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_tel">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Tel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_email">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Email</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLineEdit" name="lineEdit_nom">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_prenom"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_age"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_mobile"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_tel"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_mail"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_Adresse">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Adresse:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="adresse_textEdit"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="checkBox_newsletter">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Recevoir Sms</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_premiereVisite">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>PremiÃ¨re visite</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="dateEdit_premVisite">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_TypePeau">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Type de peau</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_type_0">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Grasse</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_type_1">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Seche</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_type_2">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Mixte</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_type_3">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Normal</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_type_4">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>CheckBox</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_sante">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <property name="text">
              <string>SantÃ©</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_sante_0">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Obj metallique</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_sante_1">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Pb cardiaque</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_sante_2">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Pb circulatoire</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_sante_3">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Alergie</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_sante_4">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Autre</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="label_info">
         <property name="text">
          <string>Infos suplÃ©mentaires</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit_info"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
